spec/models/issue_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+RSpec.describe Issue, type: :model do
+  before(:each) do
+    @issue = Issue.new(reporter_id: 321,
+                       name: 'Maria Ivanivna Polishchuk',
+                       address: '12 Chornovola street, 3, Kyiv, Ukraine',
+                       tel: '+38 (099) 22 22 4444',
```

Comment:
phone number  "+38(099)99-88-123" should also be valid
also on UI you can try to use in future some JS lib for mask input